opal/mca/accelerator/accelerator.h

```diff
@@ -111,6 +111,13 @@ struct opal_accelerator_stream_t {
 };
 typedef struct opal_accelerator_stream_t opal_accelerator_stream_t;
 
+#define MAX_IPC_HANDLE_SIZE 64
```

Comment:
I double checked, hipIpcMemHandle_t is 64 bytes, so this size is large enough, but there is also no room for error. I think at the minimum we want to confirm that 64 bytes are sufficient for cuda and ze as well.

Reply:
Thanks for checking. I copied this from libfabric https://github.com/ofiwg/libfabric/blob/main/include/ofi.h#L274

It's also used by ze so I guess it's fine. Also this var is internal to opal so we can extend it if need.